apps/src/applab/AppLabCrosshairOverlay.jsx

```diff
@@ -1,5 +1,6 @@
 /** @file App Lab-specific Crosshair Overlay */
 
+import React from 'react';
```

Comment:
Yep.  I'd like to do a followup PR that global-shims all over the place and sets up our linting to enforce it.